dashboard/scripts/archive/backfill_script_level_ids_for_teacher_feedbacks_1.rb

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
+# Backfill existing TeacherFeedbacks to set script_level_id if it can easily be # inferred because there is only one ScriptLevel associated with the level_id
+
+require_relative '../../config/environment'
+
+def feedbacks_without_script_level_id
+  TeacherFeedback.where(script_level_id: nil)
+end
+
+def puts_count
+  puts "There are #{feedbacks_without_script_level_id.count} feedbacks without a script_level_id"
+end
+
+def update_script_level_ids
+  puts "backfilling script_level_ids..."
+  feedbacks_without_script_level_id.find_each do |feedback|
+    associated_script_levels = Level.find(feedback.level_id).script_levels
```

Comment:
I think you can just:
```suggestion
    associated_script_levels = feedback.level.script_levels
```